src/Data/Number.purs

```diff
@@ -5,12 +5,34 @@ module Data.Number
   , isNaN
   , infinity
   , isFinite
+  , readInt
+  , readFloat
+  , toFixed
```

Comment:
I would rather not export `toFixed`, `toExponential`, or `toPrecision` here, since formatting numbers is already covered by Data.Number.Format.

Reply:
Agreed.